hypothesisfuzzer/fuzz_server.py

```diff
@@ -24,7 +24,9 @@ def __init__(self, config_path='config.yml'):
         self._load_config(config_path)
         self.db = SQLAlchemy(self.app)
         self.current_fuzzing_task = None
-        self.fuzzer = RepoFuzzer(self.config)
+
+        for name, repo_config in self.config['repos']:
+            self.fuzzer = RepoFuzzer(name, repo_config)
```

Comment:
Many fuzzers?  Don't quite get it plz explain.

Reply:
We want to give users the option to fuzz multiple repos in parallel. This will be an option in the config file.

Reply:
This is to support a future ticket where we allow multiple fuzzers. The second line will probably be changed to being added to a dictionary.